lib/jekyll/configuration.rb

```diff
@@ -11,7 +11,8 @@ class Configuration < Hash
       'destination'   => File.join(Dir.pwd, '_site'),
       'plugins'       => '_plugins',
       'layouts'       => '_layouts',
-      'data_source'   =>  '_data',
+      'data_source'   => '_data',
+      'includes_dir'  => '_includes',
```

Comment:
Yeppers! Examples of backwards-compatibility are in `Jekyll::Configuration`. Just add whatever is needed to that method. :)